include/mxnet/c_api.h

```diff
@@ -2317,6 +2317,11 @@ MXNET_DLL int MXNDArrayGetSharedMemHandle(NDArrayHandle handle, int* shared_pid,
 MXNET_DLL int MXNDArrayCreateFromSharedMem(int shared_pid, int shared_id, const mx_uint *shape,
                                            mx_uint ndim, int dtype, NDArrayHandle *out);
 
+/*!
+ * \brief Query cuDNN for minimum permissible epsilon for BatchNorm. If not installed, return NaN.
+ * \param result the minimum epsilon provided by cuDNN
+ */
+MXNET_DLL int MXGetCudnnBnEpsilon(float* result);
```

Comment:
should it be renamed to MXGetCudnnBNEpsilon for consistency?

Reply:
@larroy Makes sense, but given what @piiswrong  said, I'll get rid of the C call altogether and check against the 1e-5 magic value instead of adding the API that expands the macro from the particular version of `cudnn.h`.